Various patches to make Dillo happy with the web pages.

--- maradns-1.3.07.05/doc/en/webpage/dns_software.embed	2006-12-21 02:34:46.000000000 -0600
+++ maradns-1.3.07.06/doc/en/webpage/dns_software.embed	2007-09-29 21:22:11.000000000 -0500
@@ -113,7 +113,7 @@
     <p>
     I have made a
     <A href="moodns-20021011.tar.bz2">tarball available</A> for people who
-    do not want to bother with a CVS checkout.
+    do not want to bother with a CVS checkout.</p>
 </LI>
 <LI><A href="http://sourceforge.net/projects/dents">Dents</A> is a DNS
     server that showed a lot of promise.  Unfortunatly, no files have been
--- maradns-1.3.07.05/doc/en/webpage/faq-1.0.embed	2006-05-23 01:13:13.000000000 -0500
+++ maradns-1.3.07.06/doc/en/webpage/faq-1.0.embed	2007-09-29 21:30:43.000000000 -0500
@@ -39,7 +39,7 @@
 
 13. <A href="#jsstr">What string library does MaraDNS use?</A><P>
 
-14. <A href="#license">Why is MaraDNS public domain instead of BSD or GPL 
+14. <A href="#license2">Why is MaraDNS public domain instead of BSD or GPL 
     licensed?</A><P>
 
 15. <A href="#whythreads">Why does MaraDNS use a multi-threaded model?</A><P>
@@ -296,7 +296,7 @@
 are in the folder <tt>doc/man</tt> of the <A href="download.html">MaraDNS
 distribution</A>
 
-<a name="license"> </a>
+<a name="license2"> </a>
 <H2>14. Why is MaraDNS public domain instead of BSD or GPL licensed?</H2>
 
 <p>The post-1.0.xx releases of MaraDNS are, in fact, under a simple
